unittests/forked_tests_if.cpp

```diff
@@ -3,9 +3,44 @@
 #include <eosio/chain/abi_serializer.hpp>
 #include <eosio/chain/fork_database.hpp>
```

Comment:
How about renaming to `forked_tests_savanna.cpp`. I didn't do that in the rename PR to avoid conflicts with your PR.

Reply:
done